src/test/2.0/scala/com/holdenkarau/spark/testing/DataFrameSuiteBaseTest.scala

```diff
@@ -0,0 +1,78 @@
+package com.holdenkarau.spark.testing
+
+import com.holdenkarau.spark.testing.DataFrameSuiteBase.schemaErrorMessage
+import org.apache.spark.sql.Row
+import org.apache.spark.sql.types._
+import org.scalatest.FunSuite
+import org.scalatest.prop.Checkers
+
+import scala.collection.JavaConversions._
+
+case class PersonAge(name: String, age: Int)
+
+class DataFrameSuiteBaseTest
```

Comment:
Maybe we could call this DataFrameSuiteSchemaTest since this is all about testing the schema?